andela/casablanca-cfh
# #156096148 Integrate Travis CI

#### What does this PR do?
- It set up Travis CI for this project 
- It adds a Travis build status badge to the README.md
#### Description of Task to be completed?
Integrate TravisCI with readme badge
#### How should this be manually tested?
- Clone the repo.
- `cd` into the project directory.
- Run `npm install` to install dependencies.
- Make some commit.
- Push to Github.
- When you check the Travis page, a new build should be started.
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#156096148
#### Screenshots
![screencapture-github-andela-casablanca-cfh-tree-ch-integrate-travis-ci-156096148-2018-03-22-12_22_47](https://user-images.githubusercontent.com/26048536/37767836-c65568c0-2dcb-11e8-84ef-ca0d40b902cf.png)
#### Questions
None
